packages/cdk/lib/roles/cromwell-engine-role.ts

```diff
@@ -2,20 +2,46 @@ import * as cdk from "monocdk";
 import * as iam from "monocdk/aws-iam";
 import { PolicyOptions } from "../types/engine-options";
 import { BucketOperations } from "../../common/BucketOperations";
-import { S3ListAllBucketsPolicy } from "./policies/s3-list-all-buckets-policy";
+import { CromwellBatchPolicy } from "./policies/cromwell-batch-policy";
+import { Arn, ArnComponents, Stack } from "monocdk";
 
 interface CromwellEngineRoleProps {
+  account: string;
+  region: string;
   readOnlyBucketArns: string[];
   readWriteBucketArns: string[];
   policies: PolicyOptions;
+  jobQueueArn: string;
 }
 
 export class CromwellEngineRole extends iam.Role {
   constructor(scope: cdk.Construct, id: string, props: CromwellEngineRoleProps) {
+    const cromwellJobArn = Arn.format(
+      {
+        account: props.account,
```

Comment:
As per documentation at https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_core.Arn.html#static-formatcomponents-stack

```
If partition, region or account are not specified, the stack's partition, region and account will be used.
```

so we don't need to pass `region` / `account` in there, and hence we don't need them to be captured in our props too.

Reply:
Sounds good